app/templates/community/index.hbs

```diff
@@ -111,6 +111,6 @@
 
 <div class="container">
   <div class="embed-video">
-    <iframe width="854" height="480" src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0" frameborder="0" allowfullscreen></iframe>
+    <iframe width="854" height="480" src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0" frameborder="0" allowfullscreen loading="lazy"></iframe>
```

Comment:
This iframe is missing a title attribute.

Reply:
```suggestion
    <iframe width="854" height="480" src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0" frameborder="0" allowfullscreen loading="lazy" title="Ember Community Meetups, 2015"></iframe>
```